index.html

```diff
@@ -1,8 +1,60 @@
 <!DOCTYPE html>
 <html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>        
+<body>
 
-<h1>Coming soon.....</h1>
+    <div>
+        <ul class="nav-block">
+            <li class="nav-button"><a class="nav-link" href="#about-me-bkmark">About Me</a></li>
+            <li class="nav-button"><a class="nav-link" href="#portfolio-bkmark">My Portfolio</a></li>
+            <li class="nav-button"><a class="nav-link" href="#contact-bkmark">Contact Me</a></li>
+         </ul>
```

Comment:
I think that it would be best to keep your URL as clean as possible. I would remove "-bkmark" from the IDs. 

example:
`#about-me-bkmark` -> `#about-me`